stackit/internal/services/argus/instance/resource_test.go

```diff
@@ -162,6 +201,18 @@ func fixtureWebHooksConfigsGetPayload() argus.WebHook {
 	}
 }
 
+func fixtureRouteGetPayload() *argus.Route {
```

Comment:
Just noticed it now, aren't these `fixtureRouteGetResponse` instead of `Payload`?

Reply:
Yes, thats a better name. I was using `GetPayload` and `UpdatePayload`